frontend/pages/queries/QueryPage/components/QueryForm/_styles.scss

```diff
@@ -197,6 +197,19 @@
     margin-right: $pad-xsmall;
   }
 
+  &__save {
+    display: flex;
+    width: 64px;
+  }
+
+  &__save-as-new {
+    width: 82px;
+  }
+
+  &__save-modal__btn {
+    width: 106px;
+  }
+
```

Comment:
Buttons that have a loading spinner need a fixed width so they don't resize when the text is hidden and the spinner displays.